platformio.ini

```diff
@@ -12,10 +12,10 @@
 src_dir = examples/full
 lib_dir = .
 default_envs =
-	esp32dev_nimble
-	esp32dev_native
-	esp32dev
-	nano_33_iot
+	esp32dev_nimble          ;recommended 
```

Comment:
Why additional envs are commented?

Reply:
For flashing my board i commented them it was runing all environment. I  am new platformio didn't now if i can run 1 env using command or not

Reply:
Understand. Just click here:
![image](https://github.com/vovagorodok/ArduinoBleOTA/assets/8137447/6c307f70-8d42-4d32-be11-2e94675d061d)
and chose your env